--- tcl9.0.3/library/platform/platform.tcl.orig	2025-11-11 17:22:57.000000000 +0100
+++ tcl9.0.3/library/platform/platform.tcl	2025-11-11 17:23:00.000000000 +0100
@@ -275,7 +275,7 @@
     try {
 	set vdata [lindex [split [exec $libc] \n] 0]
     } on ok {} {
-	regexp {version ([0-9]+(\.[0-9]+)*)} $vdata -> v
+	regexp {version ([0-9]+(\.[0-9]+)*), by } $vdata -> v
 	lassign [split $v .] major minor
 	set v glibc${major}.${minor}
 	return 1
