--- tcl8.5.13/library/platform/platform.tcl.orig	2012-11-06 16:04:59.000000000 +0100
+++ tcl8.5.13/library/platform/platform.tcl	2013-01-01 14:14:42.116858289 +0100
@@ -256,7 +256,7 @@
     if {![catch {
 	set vdata [lindex [split [exec $libc] \n] 0]
     }]} {
-	regexp {([0-9]+(\.[0-9]+)*)} $vdata -> v
+	regexp {version ([0-9]+(\.[0-9]+)*), by } $vdata -> v
 	foreach {major minor} [split $v .] break
 	set v glibc${major}.${minor}
 	return 1
