--- tcl8.4.3/unix/mkLinks.wiget	2003-06-09 22:40:53.000000000 +0200
+++ tcl8.4.3/unix/mkLinks	2003-06-09 23:14:10.000000000 +0200
@@ -65,8 +65,8 @@
     $ZIP Access.3
     rm -f Tcl_Access.3 Tcl_Access.3.* 
     rm -f Tcl_Stat.3 Tcl_Stat.3.* 
-    ln $S Access.3$Z Tcl_Access.3$Z 
-    ln $S Access.3$Z Tcl_Stat.3$Z 
+    mv -f Access.3$Z Tcl_Access.3$Z 
+    echo '.so Tcl_Access.3' > Tcl_Stat.3
 fi
 if test -r AddErrInfo.3; then
     rm -f AddErrInfo.3.*
@@ -78,13 +78,13 @@
     rm -f Tcl_SetErrorCodeVA.3 Tcl_SetErrorCodeVA.3.* 
     rm -f Tcl_PosixError.3 Tcl_PosixError.3.* 
     rm -f Tcl_LogCommandInfo.3 Tcl_LogCommandInfo.3.* 
-    ln $S AddErrInfo.3$Z Tcl_AddObjErrorInfo.3$Z 
-    ln $S AddErrInfo.3$Z Tcl_AddErrorInfo.3$Z 
-    ln $S AddErrInfo.3$Z Tcl_SetObjErrorCode.3$Z 
-    ln $S AddErrInfo.3$Z Tcl_SetErrorCode.3$Z 
-    ln $S AddErrInfo.3$Z Tcl_SetErrorCodeVA.3$Z 
-    ln $S AddErrInfo.3$Z Tcl_PosixError.3$Z 
-    ln $S AddErrInfo.3$Z Tcl_LogCommandInfo.3$Z 
+    mv -f AddErrInfo.3$Z Tcl_AddObjErrorInfo.3$Z 
+    echo '.so Tcl_AddErrInfo.3' > Tcl_AddErrorInfo.3 
+    echo '.so Tcl_AddErrInfo.3' > Tcl_SetObjErrorCode.3
+    echo '.so Tcl_AddErrInfo.3' > Tcl_SetErrorCode.3 
+    echo '.so Tcl_AddErrInfo.3' > Tcl_SetErrorCodeVA.3
+    echo '.so Tcl_AddErrInfo.3' > Tcl_PosixError.3
+    echo '.so Tcl_AddErrInfo.3' > Tcl_LogCommandInfo.3
 fi
 if test -r Alloc.3; then
     rm -f Alloc.3.*
@@ -99,28 +99,28 @@
     rm -f ckrealloc.3 ckrealloc.3.* 
     rm -f attemptckalloc.3 attemptckalloc.3.* 
     rm -f attemptckrealloc.3 attemptckrealloc.3.* 
-    ln $S Alloc.3$Z Tcl_Alloc.3$Z 
-    ln $S Alloc.3$Z Tcl_Free.3$Z 
-    ln $S Alloc.3$Z Tcl_Realloc.3$Z 
-    ln $S Alloc.3$Z Tcl_AttemptAlloc.3$Z 
-    ln $S Alloc.3$Z Tcl_AttemptRealloc.3$Z 
-    ln $S Alloc.3$Z ckalloc.3$Z 
-    ln $S Alloc.3$Z ckfree.3$Z 
-    ln $S Alloc.3$Z ckrealloc.3$Z 
-    ln $S Alloc.3$Z attemptckalloc.3$Z 
-    ln $S Alloc.3$Z attemptckrealloc.3$Z 
+    mv -f Alloc.3$Z Tcl_Alloc.3$Z 
+    echo '.so Tcl_Alloc.3' > Tcl_Free.3 
+    echo '.so Tcl_Alloc.3' > Tcl_Realloc.3 
+    echo '.so Tcl_Alloc.3' > Tcl_AttemptAlloc.3 
+    echo '.so Tcl_Alloc.3' > Tcl_AttemptRealloc.3 
+    echo '.so Tcl_Alloc.3' > ckalloc.3 
+    echo '.so Tcl_Alloc.3' > ckfree.3 
+    echo '.so Tcl_Alloc.3' > ckrealloc.3 
+    echo '.so Tcl_Alloc.3' > attemptckalloc.3 
+    echo '.so Tcl_Alloc.3' > attemptckrealloc.3 
 fi
 if test -r AllowExc.3; then
     rm -f AllowExc.3.*
     $ZIP AllowExc.3
     rm -f Tcl_AllowExceptions.3 Tcl_AllowExceptions.3.* 
-    ln $S AllowExc.3$Z Tcl_AllowExceptions.3$Z 
+    mv -f AllowExc.3$Z Tcl_AllowExceptions.3$Z 
 fi
 if test -r AppInit.3; then
     rm -f AppInit.3.*
     $ZIP AppInit.3
     rm -f Tcl_AppInit.3 Tcl_AppInit.3.* 
-    ln $S AppInit.3$Z Tcl_AppInit.3$Z 
+    mv -f AppInit.3$Z Tcl_AppInit.3$Z 
 fi
 if test -r AssocData.3; then
     rm -f AssocData.3.*
@@ -128,9 +128,9 @@
     rm -f Tcl_GetAssocData.3 Tcl_GetAssocData.3.* 
     rm -f Tcl_SetAssocData.3 Tcl_SetAssocData.3.* 
     rm -f Tcl_DeleteAssocData.3 Tcl_DeleteAssocData.3.* 
-    ln $S AssocData.3$Z Tcl_GetAssocData.3$Z 
-    ln $S AssocData.3$Z Tcl_SetAssocData.3$Z 
-    ln $S AssocData.3$Z Tcl_DeleteAssocData.3$Z 
+    mv -f AssocData.3$Z Tcl_GetAssocData.3$Z 
+    echo '.so Tcl_AssocData.3' Tcl_SetAssocData.3
+    echo '.so Tcl_AssocData.3' Tcl_DeleteAssocData.3
 fi
 if test -r Async.3; then
     rm -f Async.3.*
@@ -140,23 +140,23 @@
     rm -f Tcl_AsyncInvoke.3 Tcl_AsyncInvoke.3.* 
     rm -f Tcl_AsyncDelete.3 Tcl_AsyncDelete.3.* 
     rm -f Tcl_AsyncReady.3 Tcl_AsyncReady.3.* 
-    ln $S Async.3$Z Tcl_AsyncCreate.3$Z 
-    ln $S Async.3$Z Tcl_AsyncMark.3$Z 
-    ln $S Async.3$Z Tcl_AsyncInvoke.3$Z 
-    ln $S Async.3$Z Tcl_AsyncDelete.3$Z 
-    ln $S Async.3$Z Tcl_AsyncReady.3$Z 
+    mv -f Async.3$Z Tcl_AsyncCreate.3$Z 
+    echo '.so Tcl_AsyncCreate.3' > Tcl_AsyncMark.3 
+    echo '.so Tcl_AsyncCreate.3' > Tcl_AsyncInvoke.3 
+    echo '.so Tcl_AsyncCreate.3' > Tcl_AsyncDelete.3 
+    echo '.so Tcl_AsyncCreate.3' > Tcl_AsyncReady.3 
 fi
 if test -r BackgdErr.3; then
     rm -f BackgdErr.3.*
     $ZIP BackgdErr.3
     rm -f Tcl_BackgroundError.3 Tcl_BackgroundError.3.* 
-    ln $S BackgdErr.3$Z Tcl_BackgroundError.3$Z 
+    mv -f BackgdErr.3$Z Tcl_BackgroundError.3$Z 
 fi
 if test -r Backslash.3; then
     rm -f Backslash.3.*
     $ZIP Backslash.3
     rm -f Tcl_Backslash.3 Tcl_Backslash.3.* 
-    ln $S Backslash.3$Z Tcl_Backslash.3$Z 
+    mv -f Backslash.3$Z Tcl_Backslash.3$Z 
 fi
 if test -r BoolObj.3; then
     rm -f BoolObj.3.*
@@ -164,9 +164,9 @@
     rm -f Tcl_NewBooleanObj.3 Tcl_NewBooleanObj.3.* 
     rm -f Tcl_SetBooleanObj.3 Tcl_SetBooleanObj.3.* 
     rm -f Tcl_GetBooleanFromObj.3 Tcl_GetBooleanFromObj.3.* 
-    ln $S BoolObj.3$Z Tcl_NewBooleanObj.3$Z 
-    ln $S BoolObj.3$Z Tcl_SetBooleanObj.3$Z 
-    ln $S BoolObj.3$Z Tcl_GetBooleanFromObj.3$Z 
+    mv -f BoolObj.3$Z Tcl_NewBooleanObj.3$Z 
+    echo '.so Tcl_NewBooleanObj.3' > Tcl_SetBooleanObj.3
+    echo '.so Tcl_NewBooleanObj.3' > Tcl_GetBooleanFromObj.3
 fi
 if test -r ByteArrObj.3; then
     rm -f ByteArrObj.3.*
