--- tcl8.0-8.0.4.orig/unix/tclUnixPipe.c
+++ tcl8.0-8.0.4/unix/tclUnixPipe.c
@@ -181,13 +181,14 @@
 				 * DString that is filled with the name of 
 				 * the temp file that was created. */
 {
-    char fileName[L_tmpnam];
+    char *fileName;
     TclFile file;
     size_t length = (contents == NULL) ? 0 : strlen(contents);
 
-    tmpnam(fileName);
-    file = TclpOpenFile(fileName, O_RDWR|O_CREAT|O_TRUNC);
+    fileName = tempnam(NULL, "tcl");
+    file = TclpOpenFile(fileName, O_RDWR|O_CREAT|O_TRUNC|O_EXCL);
     unlink(fileName);
+    free(fileName);
 
     if ((file != NULL) && (length > 0)) {
 	int fd = GetFd(file);
