--- tcl8.5a5/unix/tclUnixPort.h	2006/09/07 09:17:33	1.54
+++ tcl8.5a5/unix/tclUnixPort.h	2006/11/13 08:23:11	1.56
@@ -19,7 +19,7 @@
  * See the file "license.terms" for information on usage and redistribution
  * of this file, and for a DISCLAIMER OF ALL WARRANTIES.
  *
- * RCS: @(#) $Id$
+ * RCS: @(#) $Id$
  */
 
 #ifndef _TCLUNIXPORT
@@ -104,6 +104,12 @@
 #ifndef NO_SYS_WAIT_H
 #   include <sys/wait.h>
 #endif
+#if HAVE_INTTYPES_H
+#   include <inttypes.h>
+#endif
+#if HAVE_STDINT_H
+#   include <stdint.h>
+#endif
 #ifdef HAVE_UNISTD_H
 #   include <unistd.h>
 #else
@@ -663,6 +669,10 @@
 #include <pwd.h>
 #include <grp.h>
 
+#ifndef MODULE_SCOPE
+#define MODULE_SCOPE extern
+#endif
+
 MODULE_SCOPE struct passwd*  TclpGetPwNam(const char *name);
 MODULE_SCOPE struct group*   TclpGetGrNam(const char *name);
 MODULE_SCOPE struct passwd*  TclpGetPwUid(uid_t uid);
