diff -ur tcl8.0.5.orig/unix/configure.in tcl8.0.5/unix/configure.in
--- tcl8.0.5.orig/unix/configure.in	Sat Jun 10 20:36:38 2000
+++ tcl8.0.5/unix/configure.in	Sat Jun 10 20:39:31 2000
@@ -27,7 +27,6 @@
     CC=${CC-cc}
 AC_SUBST(CC)
 fi
-AC_C_CROSS
 
 # set the warning flags depending on whether or not we are using gcc
 if test "${GCC}" = "yes" ; then
@@ -1123,15 +1122,15 @@
 
 # Set the default compiler switches based on the --enable-symbols option
 
-AC_ARG_ENABLE(symbols, [  --enable-symbols        build with debugging symbols],
-    [tcl_ok=$enableval], [tcl_ok=no])
-if test "$tcl_ok" = "yes"; then
-    CFLAGS_DEFAULT=CFLAGS_DEBUG
-    TCL_DBGX=g
-else
+#AC_ARG_ENABLE(symbols, [  --enable-symbols        build with debugging symbols],
+#    [tcl_ok=$enableval], [tcl_ok=no])
+#if test "$tcl_ok" = "yes"; then
+#    CFLAGS_DEFAULT=CFLAGS_DEBUG
+#    TCL_DBGX=g
+#else
     CFLAGS_DEFAULT=CFLAGS_OPTIMIZE
     TCL_DBGX=""
-fi
+#fi
 
 #--------------------------------------------------------------------
 #	The statements below check for systems where POSIX-style
