--- tcl9.0.3/unix/Makefile.in.orig	2025-11-11 17:22:57.000000000 +0100
+++ tcl9.0.3/unix/Makefile.in	2025-11-11 17:23:00.000000000 +0100
@@ -105,9 +105,9 @@
 # To change the compiler switches, for example to change from optimization to
 # debugging symbols, change the following line:
 #CFLAGS			= $(CFLAGS_DEBUG)
-#CFLAGS			= $(CFLAGS_OPTIMIZE)
+CFLAGS			= $(CFLAGS_OPTIMIZE)
 #CFLAGS			= $(CFLAGS_DEBUG) $(CFLAGS_OPTIMIZE)
-CFLAGS			= @CFLAGS_DEFAULT@ @CFLAGS@
+#CFLAGS			= @CFLAGS_DEFAULT@ @CFLAGS@

 # Flags to pass to the linker
 LDFLAGS_DEBUG		= @LDFLAGS_DEBUG@
--- tcl9.0.3/unix/tcl.m4.orig	2025-11-11 17:22:57.000000000 +0100
+++ tcl9.0.3/unix/tcl.m4	2025-11-11 17:23:00.000000000 +0100
@@ -959,7 +959,7 @@
     TCL_LIB_VERSIONS_OK=ok
     CFLAGS_DEBUG=-g
     AS_IF([test "$GCC" = yes], [
-	CFLAGS_OPTIMIZE=-O2
+	CFLAGS_OPTIMIZE="${OPTFLAGS}"
 	CFLAGS_WARNING="-Wall -Wextra -Wshadow -Wundef -Wwrite-strings -Wpointer-arith"
 	case "${CC}" in
 	    *++|*++-*)
@@ -1263,7 +1263,6 @@
 	    SHLIB_CFLAGS="-fPIC -fno-common"
 	    SHLIB_SUFFIX=".so"

-	    CFLAGS_OPTIMIZE="-O2"
 	    # egcs-2.91.66 on Redhat Linux 6.0 generates lots of warnings
 	    # when you inline the string and math operations.  Turn this off to
 	    # get rid of the warnings.
