--- tcl8.6.12/unix/configure.in~	2021-10-29 19:08:08.000000000 +0200
+++ tcl8.6.12/unix/configure.in	2021-11-09 16:23:27.262943133 +0100
@@ -765,7 +765,11 @@ fi
 
 TCL_UNSHARED_LIB_SUFFIX=${UNSHARED_LIB_SUFFIX}
 TCL_SHARED_LIB_SUFFIX=${SHARED_LIB_SUFFIX}
-eval "TCL_LIB_FILE=libtcl${LIB_SUFFIX}"
+TCL_SOMAJOR=".0"
+TCL_SOVERSION="${TCL_SOMAJOR}.0"
+eval "TCL_LIB_FILE=libtcl${LIB_SUFFIX}${TCL_SOVERSION}"
+eval "TCL_LIB_SONAME=libtcl${LIB_SUFFIX}${TCL_SOMAJOR}"
+AC_SUBST(TCL_LIB_SONAME)
 
 # tclConfig.sh needs a version of the _LIB_SUFFIX that has been eval'ed
 # since on some platforms TCL_LIB_FILE contains shell escapes.
--- tcl8.6.12/unix/Makefile.in~	2021-11-09 16:23:52.000000000 +0100
+++ tcl8.6.12/unix/Makefile.in	2021-11-09 16:24:43.286276425 +0100
@@ -721,7 +721,9 @@ doc:
 # library or non-shared library for Tcl.
 ${LIB_FILE}: ${STUB_LIB_FILE} ${OBJS}
 	rm -f $@
-	@MAKE_LIB@
+	@MAKE_LIB@ -Wl,-soname,@TCL_LIB_SONAME@
+	ln -sf ${TCL_LIB_FILE} libtcl${VERSION}@SHLIB_SUFFIX@
+	ln -sf ${TCL_LIB_FILE} libtcl${VERSION}@SHLIB_SUFFIX@.0
 
 ${STUB_LIB_FILE}: ${STUB_LIB_OBJS}
 	@if test "x${LIB_FILE}" = "xlibtcl${MAJOR_VERSION}.${MINOR_VERSION}.dll"; then \

