--- tcl8.4.3/unix/Makefile.in.wiget	2003-06-09 22:21:52.000000000 +0200
+++ tcl8.4.3/unix/Makefile.in	2003-06-09 22:23:02.000000000 +0200
@@ -88,9 +88,9 @@
 # To change the compiler switches, for example to change from optimization to
 # debugging symbols, change the following line:
 #CFLAGS			= $(CFLAGS_DEBUG)
-#CFLAGS			= $(CFLAGS_OPTIMIZE)
+CFLAGS			= $(CFLAGS_OPTIMIZE)
 #CFLAGS			= $(CFLAGS_DEBUG) $(CFLAGS_OPTIMIZE)
-CFLAGS			= @CFLAGS@ @CFLAGS_DEFAULT@ -DTCL_DBGX=$(TCL_DBGX)
+#CFLAGS			= @CFLAGS@ @CFLAGS_DEFAULT@ -DTCL_DBGX=$(TCL_DBGX)
 
 # Flags to pass to the linker
 LDFLAGS_DEBUG		= @LDFLAGS_DEBUG@
