diff -Nru tcl8.0.5/generic/tclPort.h tcl8.0.5.new/generic/tclPort.h
--- tcl8.0.5/generic/tclPort.h	Mon Sep 14 20:40:01 1998
+++ tcl8.0.5.new/generic/tclPort.h	Mon Sep  6 01:12:57 1999
@@ -22,7 +22,7 @@
 #   if defined(MAC_TCL)
 #	include "tclMacPort.h"
 #    else
-#	include "../unix/tclUnixPort.h"
+#	include <tclUnixPort.h>
 #    endif
 #endif
 
diff -Nru tcl8.0.5/unix/Makefile.in tcl8.0.5.new/unix/Makefile.in
--- tcl8.0.5/unix/Makefile.in	Mon Sep  6 01:12:25 1999
+++ tcl8.0.5.new/unix/Makefile.in	Mon Sep  6 01:12:09 1999
@@ -472,9 +472,10 @@
 	    chmod +x $(SRC_DIR)/install-sh; \
 	    fi
 	@echo "Installing header files";
-	@for i in $(GENERIC_DIR)/tcl.h $(GENERIC_DIR)/tclDecls.h ; \
+	@for i in tcl.h tclDecls.h tclIntDecls.h tclMath.h tclInt.h tclPort.h tclRegexp.h ; \
 	    do \
-	    $(INSTALL_DATA) $$i $(INCLUDE_INSTALL_DIR); \
+	    $(INSTALL_DATA) $(GENERIC_DIR)/$$i $(INCLUDE_INSTALL_DIR); \
 	    done;
+	    $(INSTALL_DATA) $(UNIX_DIR)/tclUnixPort.h $(INCLUDE_INSTALL_DIR);
 	@echo "Installing library files to $(SCRIPT_INSTALL_DIR)";
 	@for i in $(TOP_DIR)/library/*.tcl $(TOP_DIR)/library/tclIndex $(UNIX_DIR)/tclAppInit.c $(UNIX_DIR)/ldAix; \
