--- tcl8.6.1/unix/Makefile.in~	2014-05-31 18:42:07.000000000 +0200
+++ tcl8.6.1/unix/Makefile.in	2014-05-31 18:42:50.235198470 +0200
@@ -176,7 +176,7 @@
 # need to modify any of these definitions by hand.
 
 STLIB_LD		= @STLIB_LD@
-SHLIB_LD		= @SHLIB_LD@
+SHLIB_LD		= @SHLIB_LD@ ${CFLAGS}
 SHLIB_CFLAGS		= @SHLIB_CFLAGS@ -DBUILD_tcl
 SHLIB_LD_LIBS		= @SHLIB_LD_LIBS@
 TCL_SHLIB_LD_EXTRAS	= @TCL_SHLIB_LD_EXTRAS@
