--- tcl8.5a2/generic/tclInt.h.orig	2005-04-21 12:21:39.460753392 +0000
+++ tcl8.5a2/generic/tclInt.h	2005-04-21 12:22:10.599019656 +0000
@@ -1957,8 +1957,10 @@
 MODULE_SCOPE void	TclpCutSockChannel _ANSI_ARGS_((Tcl_Channel chan));
 MODULE_SCOPE void	TclpSpliceFileChannel _ANSI_ARGS_((Tcl_Channel chan));
 MODULE_SCOPE void	TclpSpliceSockChannel _ANSI_ARGS_((Tcl_Channel chan));
+/* TclpPanic definition in tclUnixPort.h is... erm... weird
 MODULE_SCOPE void	TclpPanic _ANSI_ARGS_(TCL_VARARGS(CONST char *,
 			    format));
+*/
 MODULE_SCOPE char *	TclpReadlink _ANSI_ARGS_((CONST char *fileName,
 			    Tcl_DString *linkPtr));
 MODULE_SCOPE void	TclpReleaseFile _ANSI_ARGS_((TclFile file));
