--- tcl8.4.3/unix/tcl.m4.wiget	2003-06-09 23:16:10.000000000 +0200
+++ tcl8.4.3/unix/tcl.m4	2003-06-09 23:16:10.000000000 +0200
@@ -1096,7 +1096,7 @@
 	Linux*)
 	    SHLIB_CFLAGS="-fPIC"
 	    SHLIB_LD_LIBS='${LIBS}'
-	    SHLIB_SUFFIX=".so"
+	    SHLIB_SUFFIX=".so.0.0"
 
 	    # egcs-2.91.66 on Redhat Linux 6.0 generates lots of warnings 
 	    # when you inline the string and math operations.  Turn this off to
--- tcl8.4.3/unix/Makefile.in.wiget	2003-06-09 23:16:10.000000000 +0200
+++ tcl8.4.3/unix/Makefile.in	2003-06-09 23:17:12.000000000 +0200
@@ -455,7 +455,8 @@
 # a shared library or non-shared library for Tcl.
 ${LIB_FILE}: ${OBJS} ${STUB_LIB_FILE}
 	rm -f $@
-	@MAKE_LIB@
+	@MAKE_LIB@ -Wl,-soname,libtcl${VERSION}.so.0
+	-ln -sf ${TCL_LIB_FILE} libtcl${VERSION}.so
 
 ${STUB_LIB_FILE}: ${STUB_LIB_OBJS}
 	rm -f $@
