--- tcl8.3.4/unix/configure.in.orig	Sat Oct 20 02:24:15 2001
+++ tcl8.3.4/unix/configure.in	Thu Jun 27 21:43:40 2002
@@ -167,7 +167,7 @@
     AC_MSG_RESULT(yes)
 else
     AC_MSG_RESULT([broken, using substitute])
-    LIBOBJS="$LIBOBJS strstr.o"
+    AC_LIBOBJ([strstr])
 fi
 
 #--------------------------------------------------------------------
@@ -192,7 +192,7 @@
 }], , tcl_ok=0, tcl_ok=0)
 if test "$tcl_ok" = 0; then
     test -n "$verbose" && echo "	Adding strtoul.o."
-    LIBOBJS="$LIBOBJS strtoul.o"
+    AC_LIBOBJ([strtoul])
 fi
 
 #--------------------------------------------------------------------
@@ -216,7 +216,7 @@
 }], , tcl_ok=0, tcl_ok=0)
 if test "$tcl_ok" = 0; then
     test -n "$verbose" && echo "	Adding strtod.o."
-    LIBOBJS="$LIBOBJS strtod.o"
+    AC_LIBOBJ([strtod])
 fi
 
 #--------------------------------------------------------------------
@@ -297,7 +297,7 @@
     AC_CHECK_LIB(inet, strncasecmp, tcl_ok=1, tcl_ok=0)
 fi
 if test "$tcl_ok" = 0; then
-    LIBOBJS="$LIBOBJS strncasecmp.o"
+    AC_LIBOBJ([strncasecmp])
 fi
 
 #--------------------------------------------------------------------
--- tcl8.3.4/unix/tcl.m4.orig	Sat Oct 20 02:24:15 2001
+++ tcl8.3.4/unix/tcl.m4	Thu Jun 27 21:45:28 2002
@@ -682,7 +682,7 @@
 
 	    # AIX v<=4.1 has some different flags than 4.2+
 	    if test "$system" = "AIX-4.1" -o "`uname -v`" -lt "4" ; then
-		LIBOBJS="$LIBOBJS tclLoadAix.o"
+		AC_LIBOBJ([tclLoadAix])
 		DL_LIBS="-lld"
 	    fi
 
@@ -861,7 +861,7 @@
             # is kind of overkill but it works.
             # Disable inlining only when one of the
             # files in compat/*.c is being linked in.
-            if test x"${LIBOBJS}" != x ; then
+            if test x"${LIB@&t@OBJS}" != x ; then
                 EXTRA_CFLAGS="${EXTRA_CFLAGS} -fno-inline"
             fi
 
@@ -1852,7 +1852,7 @@
 	    AC_MSG_RESULT(ok)
 	else
 	    AC_MSG_RESULT(buggy)
-	    LIBOBJS="$LIBOBJS fixstrtod.o"
+	    AC_LIBOBJ([fixstrtod])
 	    AC_DEFINE(strtod, fixstrtod)
 	fi
     fi
 
 #--------------------------------------------------------------------
