--- tcl8.0/unix/configure.in.ieee	Wed Oct 22 13:39:11 1997
+++ tcl8.0/unix/configure.in	Wed Oct 22 13:43:47 1997
@@ -50,8 +50,8 @@
 #	right (and it must appear before "-lm").
 #--------------------------------------------------------------------
 
-#AC_CHECK_FUNC(sin, MATH_LIBS="", MATH_LIBS="-lm")
-#AC_CHECK_LIB(ieee, main, [MATH_LIBS="-lieee $MATH_LIBS"])
+ AC_CHECK_FUNC(sin, MATH_LIBS="", MATH_LIBS="-lm")
+ AC_CHECK_LIB(ieee, main, [MATH_LIBS="-lieee $MATH_LIBS"])
 
 #--------------------------------------------------------------------
 #       On AIX systems, libbsd.a has to be linked in to support
