--- tcl8.6.12/unix/Makefile.in~	2021-11-09 16:26:26.000000000 +0100
+++ tcl8.6.12/unix/Makefile.in	2021-11-09 16:27:13.079609680 +0100
@@ -175,7 +175,7 @@ NATIVE_TCLSH		= @TCLSH_PROG@
 # need to modify any of these definitions by hand.
 
 STLIB_LD		= @STLIB_LD@
-SHLIB_LD		= @SHLIB_LD@
+SHLIB_LD		= @SHLIB_LD@ ${CFLAGS}
 SHLIB_CFLAGS		= @SHLIB_CFLAGS@ -DBUILD_tcl
 SHLIB_LD_LIBS		= @SHLIB_LD_LIBS@
 SHLIB_LD_FLAGS		= @SHLIB_LD_FLAGS@
