diff -urN tcl8.5a2.org/unix/Makefile.in tcl8.5a2/unix/Makefile.in
--- tcl8.5a2.org/unix/Makefile.in	2004-12-08 00:16:06.000000000 +0100
+++ tcl8.5a2/unix/Makefile.in	2005-04-04 13:15:13.000000000 +0200
@@ -94,9 +94,9 @@
 # To change the compiler switches, for example to change from optimization to
 # debugging symbols, change the following line:
 #CFLAGS			= $(CFLAGS_DEBUG)
-#CFLAGS			= $(CFLAGS_OPTIMIZE)
+CFLAGS			= $(CFLAGS_OPTIMIZE)
 #CFLAGS			= $(CFLAGS_DEBUG) $(CFLAGS_OPTIMIZE)
-CFLAGS			= @CFLAGS_DEFAULT@ @CFLAGS@
+#CFLAGS			= @CFLAGS_DEFAULT@ @CFLAGS@
 
 # Flags to pass to the linker
 LDFLAGS_DEBUG		= @LDFLAGS_DEBUG@
--- tcl8.6.12/unix/tcl.m4~	2021-11-04 15:29:53.000000000 +0100
+++ tcl8.6.12/unix/tcl.m4	2021-11-09 16:19:51.296276589 +0100
@@ -1074,7 +1074,7 @@ AC_DEFUN([SC_CONFIG_CFLAGS], [
     TCL_LIB_VERSIONS_OK=ok
     CFLAGS_DEBUG=-g
     AS_IF([test "$GCC" = yes], [
-	CFLAGS_OPTIMIZE=-O2
+	CFLAGS_OPTIMIZE="${OPTFLAGS}"
 	CFLAGS_WARNING="-Wall -Wpointer-arith"
     ], [
 	CFLAGS_OPTIMIZE=-O

--- tcl8.6.12/unix/tcl.m4~	2021-11-11 14:43:00.000000000 +0100
+++ tcl8.6.12/unix/tcl.m4	2021-11-11 14:44:35.499587617 +0100
@@ -1376,7 +1376,6 @@ AC_DEFUN([SC_CONFIG_CFLAGS], [
 	    SHLIB_CFLAGS="-fPIC"
 	    SHLIB_SUFFIX=".so"
 
-	    CFLAGS_OPTIMIZE="-O2"
 	    # egcs-2.91.66 on Redhat Linux 6.0 generates lots of warnings
 	    # when you inline the string and math operations.  Turn this off to
 	    # get rid of the warnings.
