--- tcl8.4.19/unix/configure.in~	2008-08-05 17:42:26.000000000 +0300
+++ tcl8.4.19/unix/configure.in	2008-08-05 17:45:02.656851443 +0300
@@ -246,7 +246,7 @@
 	}], tcl_cv_strstr_unbroken=ok, tcl_cv_strstr_unbroken=broken,
 	    tcl_cv_strstr_unbroken=broken)])
     if test $tcl_cv_strstr_unbroken = broken; then
-        LIBOBJS="$LIBOBJS strstr.o"
+        AC_LIBOBJ([strstr])
     fi
 fi
 
@@ -278,7 +278,7 @@
     fi
 fi
 if test $tcl_ok = 0; then
-    LIBOBJS="$LIBOBJS strtoul.o"
+    AC_LIBOBJ([strtoul])
 fi
 
 #--------------------------------------------------------------------
@@ -308,7 +308,7 @@
     fi
 fi
 if test $tcl_ok = 0; then
-    LIBOBJS="$LIBOBJS strtod.o"
+    AC_LIBOBJ([strtod])
 fi
 
 #--------------------------------------------------------------------
--- tcl8.4.19/unix/tcl.m4~	2008-08-05 17:42:26.000000000 +0300
+++ tcl8.4.19/unix/tcl.m4	2008-08-05 17:46:25.053601771 +0300
@@ -952,7 +952,7 @@
 
 	    # AIX v<=4.1 has some different flags than 4.2+
 	    if test "$system" = "AIX-4.1" -o "`uname -v`" -lt "4" ; then
-		LIBOBJS="$LIBOBJS tclLoadAix.o"
+		AC_LIBOBJ([tclLoadAix])
 		DL_LIBS="-lld"
 	    fi
 
@@ -1194,7 +1194,7 @@
 	    # is kind of overkill but it works.
 	    # Disable inlining only when one of the
 	    # files in compat/*.c is being linked in.
-	    if test x"${LIBOBJS}" != x ; then
+	    if test x"${LIB@&t@OBJS}" != x ; then
 	        CFLAGS="$CFLAGS -fno-inline"
 	    fi
 
@@ -2643,7 +2643,7 @@
 		}], tcl_cv_strtod_buggy=ok, tcl_cv_strtod_buggy=buggy,
 		    tcl_cv_strtod_buggy=buggy)])
 	if test "$tcl_cv_strtod_buggy" = buggy; then
-	    LIBOBJS="$LIBOBJS fixstrtod.o"
+	    AC_LIBOBJ([fixstrtod])
 	    AC_DEFINE(strtod, fixstrtod)
 	fi
     fi
