diff -urN tcl8.5a2.org/unix/Makefile.in tcl8.5a2/unix/Makefile.in
--- tcl8.5a2.org/unix/Makefile.in	2004-12-08 00:16:06.000000000 +0100
+++ tcl8.5a2/unix/Makefile.in	2005-04-04 13:13:39.000000000 +0200
@@ -649,11 +649,11 @@
 	    chmod +x $(SRC_DIR)/install-sh; \
 	    fi
 	@echo "Installing header files";
-	@for i in $(GENERIC_DIR)/tcl.h $(GENERIC_DIR)/tclDecls.h \
-		$(GENERIC_DIR)/tclPlatDecls.h; \
+	@for i in tcl.h tclDecls.h tclIntDecls.h tclMath.h tclInt.h tclPort.h tclRegexp.h tclPlatDecls.h tclIntPlatDecls.h; \
 	    do \
-	    $(INSTALL_DATA) $$i $(INCLUDE_INSTALL_DIR); \
+	    $(INSTALL_DATA) $(GENERIC_DIR)/$$i $(INCLUDE_INSTALL_DIR); \
 	    done;
+	    $(INSTALL_DATA) $(UNIX_DIR)/tclUnixPort.h $(INCLUDE_INSTALL_DIR);
 	@echo "Installing library files to $(SCRIPT_INSTALL_DIR)";
 	@for i in $(TOP_DIR)/library/*.tcl $(TOP_DIR)/library/tclIndex $(UNIX_DIR)/tclAppInit.c $(UNIX_DIR)/ldAix; \
 	    do \
