--- tcl8.4.3/unix/configure.in.wiget	2003-05-23 02:47:23.000000000 +0200
+++ tcl8.4.3/unix/configure.in	2003-06-09 22:31:35.000000000 +0200
@@ -196,7 +196,7 @@
     AC_MSG_RESULT(yes)
 else
     AC_MSG_RESULT([broken, using substitute])
-    LIBOBJS="$LIBOBJS strstr.o"
+    AC_LIBOBJ([strstr])
 fi
 
 #--------------------------------------------------------------------
@@ -221,7 +221,7 @@
 }], , tcl_ok=0, tcl_ok=0)
 if test "$tcl_ok" = 0; then
     test -n "$verbose" && echo "	Adding strtoul.o."
-    LIBOBJS="$LIBOBJS strtoul.o"
+    AC_LIBOBJ([strtoul])
 fi
 
 #--------------------------------------------------------------------
@@ -245,7 +245,7 @@
 }], , tcl_ok=0, tcl_ok=0)
 if test "$tcl_ok" = 0; then
     test -n "$verbose" && echo "	Adding strtod.o."
-    LIBOBJS="$LIBOBJS strtod.o"
+    AC_LIBOBJ([strtod])
 fi
 
 #--------------------------------------------------------------------
@@ -327,7 +327,7 @@
     AC_CHECK_LIB(inet, strncasecmp, tcl_ok=1, tcl_ok=0)
 fi
 if test "$tcl_ok" = 0; then
-    LIBOBJS="$LIBOBJS strncasecmp.o"
+    AC_LIBOBJ([strncasecmp])
 fi
 
 #--------------------------------------------------------------------
--- tcl8.4.3/unix/tcl.m4.wiget	2003-05-23 02:47:23.000000000 +0200
+++ tcl8.4.3/unix/tcl.m4	2003-06-09 22:32:06.000000000 +0200
@@ -884,7 +884,7 @@
 
 	    # AIX v<=4.1 has some different flags than 4.2+
 	    if test "$system" = "AIX-4.1" -o "`uname -v`" -lt "4" ; then
-		LIBOBJS="$LIBOBJS tclLoadAix.o"
+		AC_LIBOBJ([tclLoadAix])
 		DL_LIBS="-lld"
 	    fi
 
@@ -1131,7 +1131,7 @@
 	    # is kind of overkill but it works.
 	    # Disable inlining only when one of the
 	    # files in compat/*.c is being linked in.
-	    if test x"${LIBOBJS}" != x ; then
+	    if test x"${LIB@&t@OBJS}" != x ; then
 	        EXTRA_CFLAGS="${EXTRA_CFLAGS} -fno-inline"
 	    fi
 
@@ -2243,7 +2243,7 @@
 	    AC_MSG_RESULT(ok)
 	else
 	    AC_MSG_RESULT(buggy)
-	    LIBOBJS="$LIBOBJS fixstrtod.o"
+	    AC_LIBOBJ([fixstrtod])
 	    AC_DEFINE(strtod, fixstrtod)
 	fi
     fi
