--- tcl8.4.6/unix/configure.in.orig	2004-03-06 20:29:26.000000000 +0100
+++ tcl8.4.6/unix/configure.in	2004-03-06 21:47:35.627341128 +0100
@@ -452,7 +452,11 @@
 
 TCL_UNSHARED_LIB_SUFFIX=${UNSHARED_LIB_SUFFIX}
 TCL_SHARED_LIB_SUFFIX=${SHARED_LIB_SUFFIX}
-eval "TCL_LIB_FILE=libtcl${LIB_SUFFIX}"
+TCL_SOMAJOR=".0"
+TCL_SOVERSION="${TCL_SOMAJOR}.0"
+eval "TCL_LIB_FILE=libtcl${LIB_SUFFIX}${TCL_SOVERSION}"
+eval "TCL_LIB_SONAME=libtcl${LIB_SUFFIX}${TCL_SOMAJOR}"
+AC_SUBST(TCL_LIB_SONAME)
 
 SC_ENABLE_FRAMEWORK
 
--- tcl8.4.6/unix/Makefile.in.orig	2004-03-06 20:29:26.000000000 +0100
+++ tcl8.4.6/unix/Makefile.in	2004-03-06 21:54:37.004282096 +0100
@@ -455,7 +455,8 @@
 # a shared library or non-shared library for Tcl.
 ${LIB_FILE}: ${OBJS} ${STUB_LIB_FILE}
 	rm -f $@
-	@MAKE_LIB@
+	@MAKE_LIB@ -Wl,-soname,@TCL_LIB_SONAME@
+	-ln -sf ${TCL_LIB_FILE} libtcl${VERSION}@SHLIB_SUFFIX@
 
 ${STUB_LIB_FILE}: ${STUB_LIB_OBJS}
 	rm -f $@
