--- tcl9.0.3/library/init.tcl.orig	2025-11-11 17:22:57.000000000 +0100
+++ tcl9.0.3/library/init.tcl	2025-11-11 17:23:00.000000000 +0100
@@ -60,16 +60,11 @@
 namespace eval tcl {
     if {![interp issafe]} {
 	variable Dir
-	foreach Dir [list $::tcl_library [file dirname $::tcl_library]] {
+	foreach Dir [list $::tcl_library] {
 	    if {$Dir ni $::auto_path} {
 		lappend ::auto_path $Dir
 	    }
 	}
-	set Dir [file join [file dirname [file dirname \
-		[info nameofexecutable]]] lib]
-	if {$Dir ni $::auto_path} {
-	    lappend ::auto_path $Dir
-	}
 	if {[info exists ::tcl_pkgPath]} { catch {
 	    foreach Dir $::tcl_pkgPath {
 		if {$Dir ni $::auto_path} {
