--- tcl8.6.16/tests/cmdAH.test.orig	2025-03-16 09:49:13.533013671 +0100
+++ tcl8.6.16/tests/cmdAH.test	2025-03-16 09:50:32.705918088 +0100
@@ -1311,22 +1311,6 @@ test cmdAH-24.14.1 {
     file mtime [file join [temporaryDirectory] CON.txt]
 } -match regexp -result {could not (?:get modification time|read)} -returnCodes error
 
-# 3155760000 is 64-bit Unix time, Wed Jan 01 00:00:00 GMT 2070:
-test cmdAH-24.20.1 {Tcl_FileObjCmd: atime 64-bit time_t, bug [4718b41c56]} -constraints {time64bit filetime64bit} -setup {
-    set filename [makeFile "" foo.text]
-} -body {
-    list [file atime $filename 3155760000] [file atime $filename]
-} -cleanup {
-    removeFile $filename
-} -result {3155760000 3155760000}
-test cmdAH-24.20.2 {Tcl_FileObjCmd: mtime 64-bit time_t, bug [4718b41c56]} -constraints {time64bit filetime64bit} -setup {
-    set filename [makeFile "" foo.text]
-} -body {
-    list [file mtime $filename 3155760000] [file mtime $filename]
-} -cleanup {
-    file delete -force $filename
-} -result {3155760000 3155760000}
-
 # owned
 test cmdAH-25.1 {Tcl_FileObjCmd: owned} -returnCodes error -body {
     file owned a b
